kazoo/handlers/utils.py

```diff
@@ -122,6 +113,18 @@ def unlink(self, callback):
             if callback in self._callbacks:
                 self._callbacks.remove(callback)
 
+    def _do_callbacks(self):
```

Comment:
Can you please add a little description to this method? Just to help make the read straightforward for any newcomer.

Reply:
Sure. I'll add one.

Reply:
Done.